### DISCUSSION TITLE
Osu!Lazer Mobile (Android) Can't Login, No IPV6

### DISCUSSION THREAD

**Discussion by apowersak1** [ppy/osu#14522] [Q&A]
**Describe the bug:** Can't login, will not authenticate, tried to use Cloudflare 1.1.1.1, couldn't authenticate still, In Canada, not sure if that has anything to do with it, can login normally on the website and on PC

**Screenshots or videos showing encountered issue:** 

**osu!lazer version:** Latest Release, downloaded straight from Github

**Logs:** 2021-08-27 06:58:00 [verbose]: ---------------------------------------------------------- 2021-08-27 06:58:00 [verbose]: network Log for somebody (LogLevel: Verbose) 2021-08-27 06:58:00 [verbose]: Running 2021.819.0.0 on .NET 4.0.50524.0 2021-08-27 06:58:00 [verbose]: Environment: Android (Unix 4.9.186.18961046), 4 cores 2021-08-27 06:58:00 [verbose]: ---------------------------------------------------------- 2021-08-27 06:58:00 [verbose]: Request to https://api.github.com/repos/ppy/osu/releases/latest successfully completed! 2021-08-27 06:58:21 [verbose]: Request to https://osu.ppy.sh/oauth/token failed with System.Net.WebException: BadRequest. 2021-08-27 06:58:21 [verbose]: Response was: {"error":"The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client."} 2021-08-27 06:58:21 [verbose]: Login failed!

**Comment by peppy** [comment]
The error is saying that the token is invalid. Are you sure you are using your username to login, not email?
